config/routes.rb

```diff
@@ -247,4 +239,18 @@
 
   # Startpage
   root :to => 'home#show'
+
+  #OpenID Connect & OAuth
+  namespace :openid_connect do
+    resources :clients, only: :create
+    post 'access_tokens', to: proc { |env| OpenidConnect::TokenEndpoint.new.call(env) }
+
+    # Authorization Servers MUST support the use of the HTTP GET and POST methods at the Authorization Endpoint (see http://openid.net/specs/openid-connect-core-1_0.html#AuthResponseValidation).
+    resources :authorizations, only: [:new, :create]
+    post 'authorizations/new', to: 'authorizations#new'
+  end
+
+  api_version(module: "Api::V0", path: {value: "api/v0"}, default: true) do
+    match 'user', to: 'users#show', via: [:get, :post]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use %i or %I for array of symbols.